server/ai_mediaserver.go

```diff
@@ -389,6 +389,14 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 		}
 		ctx = clog.AddVal(ctx, "source_type", sourceType)
 
+		remoteHost, err := getRemoteHost(r.RemoteAddr)
```

Comment:
This is OK if it gets us going for now. 

I can see more sophisticated setups where this might not really work, eg proxies or if RTMP is listening on a nonstandard port. For those cases I would expect another form parameter in the curl (including possibly a separate field for the control URL if its also on a nonstandard port)

Reply:
Yep I guess we can configure an environment variable in the mediamtx pod for the callback hostname, then that would be available to the curl command.